chatbot/src/test/java/chatbot/core/handlers/rivescript/RiveScriptQueryHandlerTest.java

```diff
@@ -97,8 +93,6 @@ public void testSearchDbpedia() throws IOException {
 		String expectedHelloOutput = "I am the DBpedia Bot";
 		log.info("In Dbpedia Test case");
```

Comment:
@Juzerhk Can you also please remove these log.XXX lines. They don't add much to debugging even if an error occurs. After removing please also remove Log object initialisation and also unused imports related to log (Check how to organise imports in eclipse).

Reply:
done